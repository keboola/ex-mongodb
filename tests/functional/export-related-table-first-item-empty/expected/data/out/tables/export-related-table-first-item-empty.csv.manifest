{"incremental":false,"columns":["id"],"primary_key":["id"],"column_metadata":{"id":[{"key":"KBC.datatype.nullable","value":true}]}}