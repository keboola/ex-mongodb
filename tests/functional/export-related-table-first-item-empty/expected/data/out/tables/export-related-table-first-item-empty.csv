"5716054cee6e764c94fadb23"
"5716054cee6e764c94fadb22"
"5716054cee6e764c94fadb21"
